package.json

```diff
@@ -90,13 +90,22 @@
     "text-mask-addons": "^3.8.0"
   },
   "peerDependencies": {
+    "@emotion/react": "^11.9.3",
+    "@emotion/styled": "^11.9.3",
     "@material-ui/core": ">= 4 < 5",
     "@material-ui/icons": ">= 4 < 5",
     "@material-ui/lab": "^4.0.0-alpha.46",
     "@material-ui/pickers": "^4.0.0-alpha.12",
+    "@mui/icons-material": "^5.8.4",
+    "@mui/lab": "^5.0.0-alpha.92",
+    "@mui/x-date-pickers": "^5.0.0-beta.2",
+    "@mui/material": "^5.9.2",
+    "@mui/styles": "^5.9.2",
+    "ag-grid-community": "^25.2.1",
+    "ag-grid-react": "^25.2.0",
     "@selectquotelabs/sqhooks": ">= 2",
     "moment": "^2.29.1",
-    "react": ">= 16.8.0 < 18",
+    "react": ">= 17.0.0 < 18",
     "react-dom": ">= 16.8.0 < 18",
     "scplus-shared-components": ">= 7",
```

Comment:
This should be updated to >= 10 now

Reply:
Are you referring to scplus-shared-components?  We want the react versions to be no less that v17?

Reply:
Yep! Github probably being weird and putting my comment as showing on a different line.